src/main/include/utility/MotorUtils.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <frc2/command/SubsystemBase.h>
+#include <frc/smartdashboard/SmartDashboard.h>
+#include "ctre/phoenix6/TalonFX.hpp"
+
+#include <vector>
+#include <string>
+
+namespace MotorUtils {
+  struct PIDValues {
+    double kS = 0;
+    double kP = 0;
+    double kI = 0;
+    double kD = 0;
+    double kG = 0;
+  };
+
+  struct Motor {
+    struct LogValues {
+      bool position = false;
+      bool velocity = false;
+      bool acceleration = false;
+    };
+
+    ctre::phoenix6::hardware::TalonFX *motorPtr;
+    std::string name;
```

Comment:
Consider setting name to the name given to the device in TunerX via `.GetDescription()`